CHANGELOG.md

```diff
@@ -15,7 +15,9 @@
 
 ##### Bug Fixes
 
-* None.  
+* Add support for `runOncePerArchitecture` attribute for `PBXBuildRule`.
```

Comment:
nit: please add two empty spaces after this sentence for markdown formatting.

Reply:
Done